spec/models/candidate_interface/gcse_qualification_details_form_spec.rb

```diff
@@ -4,6 +4,30 @@
   describe 'validations' do
     it { is_expected.to validate_presence_of(:grade) }
     it { is_expected.to validate_presence_of(:award_year) }
+
+    describe 'grade format' do
```

Comment:
We should add a test for each of the different regexes. At the mimimum test that [Juan's examples from the document](https://docs.google.com/document/d/1NMUgKmzPSWJn85CVCUjkeArGuy1_WwlVeFZ9QNy6AeA/edit) pass.

Reply:
Done!